twisted/words/protocols/jabber/client.py

```diff
@@ -160,15 +160,15 @@ class BasicAuthenticator(xmlstream.ConnectAuthenticator):
     U{JEP-0077<http://www.jabber.org/jeps/jep-0077.html>}.
 
     Under normal circumstances, the BasicAuthenticator generates the
-    L{xmlstream.STREAM_AUTHD_EVENT} once the stream has authenticated. However,
+    C{xmlstream.STREAM_AUTHD_EVENT} once the stream has authenticated. However,
```

Comment:
since this is public API I think that we should expose the documentation for STREAM_AUTHD_EVENT and link to it